docs/src/pages/customization/css-in-js.md

```diff
@@ -64,6 +64,12 @@ They are easy to debug in development and as short as possible in production:
 - development: `.MuiAppBar-root-12`
 - production: `.c12`
 
+If you don't like this default behavior, you can change it.
+JSS relies on the concept of [class name generator](http://cssinjs.org/js-api/#generate-your-own-class-names).
+We provide a custom implementation for Material-UI needs:
+[`createGenerateClassName()`](#creategenerateclassname-options-class-name-generator).
+As well as the option to make the class names **deterministic** with the `dangerouslyUseGlobalCSS` option.
```

Comment:
Maybe worth adding that class names may change without major version change?

Reply:
Why would they change without major version change?

Reply:
actually, if we are not changing components name and rule names are anyways already public through classes override, it doesn't matter